draft-irtf-cfrg-frost.md

```diff
@@ -507,11 +511,14 @@ below, and their corresponding signing key shares.
 In particular, it is assumed that the coordinator and each signing participant `P_i` knows the following
 group info:
 
-- Group public key, an `Element` in `G`, denoted `PK = G.ScalarMultBase(s)`, corresponding to the group secret key `s`, which is a `Scalar` in `GF(p)`. `PK` is an output from the group's key generation protocol, such as `trusted_dealer_keygen`or a DKG.
+- Group public key, an `Element` in `G`, denoted `PK = G.ScalarMultBase(s)`, corresponding
+  to the group secret key `s`, which is a `Scalar` in `GF(p)`. `PK` is an output from the group's
+  key generation protocol, such as `trusted_dealer_keygen`or a DKG.
 - Public keys for each signer, denoted `PK_i = G.ScalarMultBase()`, which are similarly
-outputs from the group's key generation protocol, `Element`s in `G`.
+  outputs from the group's key generation protocol, `Element`s in `G`.
 
-And that each participant with identifier `i`  where `i` is an integer in the range between 1 and NUM_SIGNERS additionally knows the following:
+And that each participant with identifier `i`  where `i` is an integer in the range between 1
+and MAX_SIGNERS additionally knows the following:
```

Comment:
Hm. This should be NUM_SIGNERS, no? I.e, if THRESHOLD_LIMIT=t, NUM_SIGNERS=n, then MAX_SIGNERS is at least t but at most n?